Gems/Atom/RHI/Vulkan/Code/Source/RHI.Reflect/Conversion.cpp

```diff
@@ -165,8 +165,8 @@ namespace AZ
                 return VK_QUEUE_GRAPHICS_BIT;
             case RHI::HardwareQueueClass::Compute:
                 return VK_QUEUE_COMPUTE_BIT;
-            case RHI::HardwareQueueClass::Copy:                
-                return VK_QUEUE_TRANSFER_BIT;
+            case RHI::HardwareQueueClass::Copy:
+                return VK_QUEUE_TRANSFER_BIT|VK_QUEUE_SPARSE_BINDING_BIT;
```

Comment:
Do we need to check device capabilities before adding this bit here?

Reply:
So the ConvertQueueClass function is only used by CommandQueueContext::SelectQueueFamily which it will filter out unsupported flags.